test/e2e/wicd_rbac_test.go

```diff
@@ -0,0 +1,139 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	certificatesv1 "k8s.io/api/certificates/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/windows-machine-config-operator/pkg/rbac"
+)
+
+// wicdRBACTestSuite tests WICD RBAC manifests and certificate functionality
+func wicdRBACTestSuite(t *testing.T) {
+	testCtx, err := NewTestContext()
+	if err != nil {
+		t.Fatalf("Failed to create test context: %v", err)
+	}
+
+	// Get all Windows nodes to determine if RBAC is relevant
+	windowsNodes, err := testCtx.client.K8s.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{
```

Comment:
This is going to pick up Nodes not created by the test suite. Please use one of the existing helpers